### PR TITLE
Feature 54543 rails frontend styling

### PR DESCRIPTION
**Pull request by LBruegger**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -30,9 +30,10 @@
     "ember-auto-import": "^1.5.3",
     "ember-autofocus-modifier": "^1.0.1",
     "ember-bootstrap": "^3.1.4",
+    "ember-bootstrap-power-select": "^2.0.0",
     "ember-changeset": "^3.3.8",
     "ember-changeset-validations": "^3.3.2",
-    "ember-cli": "~3.16.2",
+    "ember-cli": "~3.19.0",
```
Comment (TheWalkingLeek) [inline frontend/package.json:36]: Huh? Why is this here? Also, there's a bug of powerselect in 3.19. I'd rather we stay on 3.16 since it's also LTS

File: frontend/app/templates/components/account-form.hbs [hunk 2]
```diff
@@ -4,68 +4,76 @@
   </Modal.header>
```
Comment (TheWalkingLeek) [inline frontend/app/templates/components/account-form.hbs:4]: Why was this file even touched here? Looks like that Bootstrap stuff... Is this here on purpose?

File: frontend/app/templates/components/nav-bar.hbs [hunk 3]
```diff
@@ -21,7 +21,7 @@
         {{/if}}
```
Comment (TheWalkingLeek) [inline frontend/app/templates/components/nav-bar.hbs:21]: Again, I don't see why this file is even touched. This story is purposefully just for the SSR Frontend.

File: frontend/app/templates/components/account-form.hbs (hunk 2 above)
Comment (LBruegger) [reply frontend/app/templates/components/account-form.hbs:4]: Merge conflict.

File: frontend/app/templates/components/nav-bar.hbs (hunk 3 above)
Comment (LBruegger) [reply frontend/app/templates/components/nav-bar.hbs:21]: Again Merge Conflict.

**Comment by LBruegger** [comment]
> image
> 
> I currently only get the translation key when changing a user role. Also, the table header "Admin" doesn't seem right there, shouldn't it be role?

This is not my fault. Still there on master. But I will fix it for Sure.

**Comment by LBruegger** [comment]
> almost there ...
> 
> * if I only select a team, I cannot select any folders from the drop down in the account form. the folder field also has errors then.
> * if only a team is selected, it should pre-select this team, but no errors yet for the folder. the user has to select a folder first.
> 
> code looks good !

I just fixed the Collapsable issues from Folder and Team. This is now not an issue anymore 😉

**Comment by psunix** [comment]
image

since bob is not a ConfAdmin nor a Admin, the Settings dropdown should not be displayed.
